src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -23,10 +23,17 @@ def get(self, request, project):
         # TODO: conditions need to be based on actions
         for rule_type, rule_cls in rules:
             node = rule_cls(project)
-            context = {"id": node.id, "label": node.label, "enabled": node.is_enabled()}
+            context = {
+                "id": node.id,
+                "label": node.label,
+                "enabled": node.is_enabled(),
+            }
             if hasattr(node, "prompt"):
                 context["prompt"] = node.prompt
 
+            if hasattr(node, "category"):
+                context["category"] = node.category
```

Comment:
yup, since actions don't have the category field. Now that I look at this again though I'm wondering if I should just use the existing `rule_type` field and set it to something like `filter/event` for filters instead of adding a `category` field. Would that be preferred or is `rule_type` not meant for something like that?

Reply:
That might actually make more sense. You'll need to check all the places we look for `rule_type` though, not sure if it's used on the frontend as well.

Reply:
yea i think ill go for that, i dont want the names to get too weird with an `EventCondition` being actually a condition or a filter and having to differentiate that with another field. I did a grep for `rule_type` and it mainly seems to be used for distinguishing between actions and conditions. There is a mention of it in `ProjectAgnosticRuleConditionsEndpoint` but I think we can leave that as is.